packages/cli/src/write-script/render-foundry.ts

```diff
@@ -0,0 +1,74 @@
+import { EOL } from 'node:os';
+import { Transform } from 'node:stream';
+
+/**
+ * This script is used to deploy contracts using Foundry Cast.
+ * It outputs a bash script that can be used to deploy contracts and execute transactions.
+ * Note: Make sure you add `.sh` extension to the output file.
+ */
+
+const header = `#!/bin/bash
+
+# Set your environment variables
+export PRIVATE_KEY=<your_private_key>
+export RPC_URL=<your_rpc_url>
```

Comment:
This 2 lines should have working values so the script does not need to be modified to work, e.g. it can be easily generated during CI and just add the 2 env vars:
```suggestion
export PRIVATE_KEY=${PRIVATE_KEY:-""}
export RPC_URL=${RPC_URL:-""}
```